test/Card-test.js

```diff
@@ -2,31 +2,32 @@ const chai = require('chai');
 const expect = chai.expect;
 
 const Card = require('../src/Card');
+const data = require('../src/data');
```

Comment:
Not sure this is actually necessary, but I forgot to delete it after I added it while chasing my tail in confused circles for a while. Probably best to delete it since it doesn't seem like it changed anything.